utils/python/CIME/XML/generic_xml.py

```diff
@@ -227,7 +227,9 @@ def get_resolved_value(self, raw_value):
                 srcroot = os.path.join(get_cime_root(),"..")
                 item_data = item_data.replace(m.group(), srcroot)
             elif var in os.environ:
-                logging.warn("Resolved from env: " + var)
+                # this is a list of suppressed warnings (things normally expected to be resolved in env)
+                if var not in ("USER"):
```

Comment:
This is definitely wrong. You want 
```
("USER",)
```
... if you want that to be interpreted as a tuple.